acq4/devices/PMT/PMT.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+from acq4.devices.Device import *
+from acq4.devices.OptomechDevice import OptomechDevice
+from acq4.devices.DAQGeneric import DAQGeneric
+
+class PMT(DAQGeneric, OptomechDevice):
```

Comment:
I am only suggesting changing the class name on line 6 to `PhotoDetector`, and renaming the file/directory accordingly. There should be no need to change anything else in Imager or elsewhere in ACQ4, and you can still name your device "PMT" in the configuration file:

    PMT:
        driver: "PhotoDetector"

Maybe I still don't understand the question?